controllers/reviews.controllers.js

```diff
@@ -38,9 +39,16 @@ exports.getCommentsByReviewId = (req, res, next) => {
 };
 
 exports.getReviews = (req, res, next) => {
-  fetchReviews(req.query.category)
-    .then((reviews) => {
-      res.status(200).send(reviews);
+  const { category, sort_by, order } = req.query;
+  const promises = [fetchReviews(category, sort_by, order)];
+
+  if (category) {
+    promises.push(fetchCategoryBySlug(category));
```

Comment:
As fetchCategoryBySlug is a utility function we should really be unit testing this. You would have to query the database in your tests to confirm that this works as expected.